src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -654,7 +662,7 @@ inadvertently reveal passwords.
 You can see both ARIA features in action in the following Glitch:
 
 {% Glitch {
-  id: 'signin-form',
+  id: 'sign-in-form',
```

Comment:
The other Glitch exists as well, but I updated everything to sign-in instead of signin.